packages/jellyfish-transaction-builder/src/txn/txn_builder_account.ts

```diff
@@ -124,6 +124,21 @@ export class TxnBuilderAccount extends P2WPKHTxnBuilder {
     )
   }
 
+  /**
+   * TransferDomain transaction.
+   *
+   * @param {TransferDomain} transferDomain txn to create
+   * @param {Script} changeScript to send unspent to after deducting the (converted + fees)
+   * @returns {Promise<TransactionSegWit>}
+   */
+
+  async transferDomain (transferDomain: TransferDomain, changeScript: Script): Promise<TransactionSegWit> {
```

Comment:
No tests for this?

Reply:
split the `txn-builder` to another PR first